src/modules/Elsa.MassTransit.AzureServiceBus/Features/AzureServiceBusFeature.cs

```diff
@@ -31,18 +53,81 @@ public override void Configure()
         {
             massTransitFeature.BusConfigurator = configure =>
             {
+                var consumers = massTransitFeature.GetConsumers().ToList();
+                var shortLivedConsumers = consumers
```

Comment:
The term "temporary" or even "temporaryQueue" seems more concise.
E.g. `temporaryConsumers` or `temporaryQueueConsumers`. The latter is also more correct, given that it's not the consumers that are short-live/temporary, but the queues are.